src/main/java/org/opensearch/security/resolver/IndexResolverReplacer.java

```diff
@@ -309,7 +309,7 @@ private void resolveIndexPatterns(
                 final boolean isDebugEnabled = log.isDebugEnabled();
                 try {
                     matchingAllIndices = Arrays.asList(
-                        resolver.concreteIndexNames(state, indicesOptions, localRequestedPatterns.toArray(new String[0]))
+                        resolver.concreteIndexNames(state, indicesOptions, true, localRequestedPatterns.toArray(new String[0]))
```

Comment:
This change is needed to ensure that the concrete index corresponding to any data streams in the request is resolved to.

Reply:
Do we also want to set this as true in line 489/is there a reason this was not passed as true before?